networks/afnonet.py

```diff
@@ -191,14 +191,14 @@ def __init__(
         self.patch_size = (params.patch_size, params.patch_size)
         self.in_chans = params.N_in_channels
         self.out_chans = params.N_out_channels
-        self.num_features = self.embed_dim = embed_dim
+        self.num_features = self.embed_dim = params.embed_dim if 'embed_dim' in params else embed_dim
```

Comment:
This line checks is embed_dim is specified in params. If not, then use whatever AFNONet was initialized with (most likely, the default value of 768). I guess the other option is to change where AFNONet is initialized and pass params.embed_dim there. Neither are great options so I think this is fine.

Reply:
Yeah, I chose this option since it is consistent with how other parameters like `patch_size` and `img_size` are set (via things in `params` instead of using the kwargs provided to AFNONet's init. Agreed it's not ideal—those kwargs are ignored—but I think consistency is worth it.